### PR TITLE
Convert cheatsheet to JSON: Gimp

### PR DESCRIPTION
**Pull request by kbhat95**
This is in reference to #1155 .

One problem, as I mentioned in #1155, is that we have subsections in the old cheatsheet. Although strictly not necessary, I think they make it look better. How might I go about putting that in (if I need to)?

### CODE REVIEW

**Comment by moollaza** [comment]
>we have subsections in the old cheatsheet. Although strictly not necessary, I think they make it look better. How might I go about putting that in (if I need to)?

We don't support anything like this yet, I'm not sure it will be possible given that the sections re-order themselves to fit. I'd suggest you skup the sub-sections, or break them out into their own sections.

I'll play around with the title's to see how it looks and get back to you.

**Comment by moollaza** [comment]
I think the current design/sections look fine :+1:

File: share/goodie/cheat_sheets/gimp.json [hunk 1]
```diff
@@ -0,0 +1,462 @@
...
+                "val": "Path to selection"
+            }
+        ],
+        "Layers": [
+            {
+                "key": "[PgUp] or [Ctrl + Tab]",
+                "val": "Select the layer above"
+            },
+            {
+                "key": "[PgDn]\n or\n [Shift + Ctrl + Tab]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/gimp.json]: no newlines chars please :) 

this should be:
`[PgDn] or [Shift] + [Ctrl] + [Tab]`

File: share/goodie/cheat_sheets/gimp.json [hunk 2]
```diff
@@ -0,0 +1,462 @@
...
+                "val": "Float selection"
+            },
+            {
+                "key": "[Shift] + [V]",
+                "val": "Path to selection"
+            }
+        ],
+        "Layers": [
+            {
+                "key": "[PgUp] or [Ctrl + Tab]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/gimp.json]: `[PgUp] or [Ctrl] + [Tab]`

File: share/goodie/cheat_sheets/gimp.json [hunk 3]
```diff
@@ -0,0 +1,462 @@
...
+            {
+                "key": "[Shift] + [Ctrl] + [P]",
+                "val": "Patterns"
+            },
+            {
+                "key": "[Ctrl] + [G]",
+                "val": "Gradients"
+            },
+            {
+                "key": "[Alt + F4] or [Ctrl + W]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/gimp.json]: `[Alt] + [F4] or [Ctrl] + [W]`

File: share/goodie/cheat_sheets/gimp.json [hunk 4]
```diff
@@ -0,0 +1,462 @@
...
+            {
+                "key": "Enter",
+                "val": "Set the new value"
+            },
+            {
+                "key": "[Space] or [Enter]",
+                "val": "Activate current button or list"
+            },
+            {
+                "key": "[Ctrl + Alt + PgUp] and [Ctrl + Alt + PgDn]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/gimp.json]: [Ctrl] + [Alt] + [PgUp] / [ PgDn]

File: share/goodie/cheat_sheets/gimp.json [hunk 5]
```diff
@@ -0,0 +1,462 @@
...
+                "val": "Close Dialog"
+            }
+        ],
+        "View": [
+            {
+                "key": "F10",
+                "val": "Main Menu"
+            },
+            {
+                "key": "[Shift + F10] or [right click]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/gimp.json]: `[Shift] + [F10] or [right click]`

**Comment by kbhat95** [comment]
@moollaza, I hope it looks better now!

**Comment by moollaza** [comment]
LGTM :+1:

**Comment by moollaza** [comment]
@kbhat95 congrats! It's live: https://duckduckgo.com/?q=gimp+cheat+sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**